Doc/whatsnew/3.13.rst

```diff
@@ -1049,6 +1049,95 @@ Deprecated
+
+  The :c:func:`Py_InitializeFromConfig` API should be used with
+  :c:type:`PyConfig` instead.
+
+* Creating :c:data:`immutable types <Py_TPFLAGS_IMMUTABLETYPE>` with mutable
+  bases (:gh:`95388`).
+
+Pending Removal in Python 3.15
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* :c:func:`PyImport_ImportModuleNoBlock`: use :c:func:`PyImport_ImportModule`
+* :c:func:`PyWeakref_GET_OBJECT`: use :c:func:`PyWeakref_GetRef`
+* :c:func:`PyWeakref_GetObject`: use :c:func:`PyWeakref_GetRef`
+* :c:type:`!Py_UNICODE_WIDE` type: use :c:type:`wchar_t`
+* :c:type:`Py_UNICODE` type: use :c:type:`wchar_t`
+* Python initialization functions:
+
+  * :c:func:`PySys_ResetWarnOptions`: clear :data:`sys.warnoptions` and
+    :data:`!warnings.filters`
+  * :c:func:`Py_GetExecPrefix`: get :data:`sys.exec_prefix`
+  * :c:func:`Py_GetPath`: get :data:`sys.path`
+  * :c:func:`Py_GetPrefix`: get :data:`sys.prefix`
+  * :c:func:`Py_GetProgramFullPath`: get :data:`sys.executable`
+  * :c:func:`Py_GetProgramName`: get :data:`sys.executable`
+  * :c:func:`Py_GetPythonHome`: get :c:member:`PyConfig.home` or
+    the :envvar:`PYTHONHOME` environment variable
+
+Pending Removal in Future Versions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following APIs are deprecated and will be removed,
+although there is currently no date scheduled for their removal.
+
+* :c:macro:`Py_TPFLAGS_HAVE_FINALIZE`: unneeded since Python 3.8
+* :c:func:`PyErr_Fetch`: use :c:func:`PyErr_GetRaisedException`
+* :c:func:`PyErr_NormalizeException`: use :c:func:`PyErr_GetRaisedException`
+* :c:func:`PyErr_Restore`: use :c:func:`PyErr_SetRaisedException`
+* :c:func:`PyModule_GetFilename`: use :c:func:`PyModule_GetFilenameObject`
+* :c:func:`PyOS_AfterFork`: use :c:func:`PyOS_AfterFork_Child`
+* :c:func:`PySlice_GetIndicesEx`: use :c:func:`PySlice_Unpack` and :c:func:`PySlice_AdjustIndices`
+* :c:func:`!PyUnicode_AsDecodedObject`: use :c:func:`PyCodec_Decode`
+* :c:func:`!PyUnicode_AsDecodedUnicode`: use :c:func:`PyCodec_Decode`
+* :c:func:`!PyUnicode_AsEncodedObject`: use :c:func:`PyCodec_Encode`
+* :c:func:`!PyUnicode_AsEncodedUnicode`: use :c:func:`PyCodec_Encode`
+* :c:func:`PyUnicode_READY`: unneeded since Python 3.12
+* :c:func:`!PyErr_Display`: use :c:func:`PyErr_DisplayException`
+* :c:func:`!_PyErr_ChainExceptions`: use ``_PyErr_ChainExceptions1``
+* :c:member:`!PyBytesObject.ob_shash` member:
+  call :c:func:`PyObject_Hash` instead
+* :c:member:`!PyDictObject.ma_version_tag` member
+* Thread Local Storage (TLS) API:
+
+  * :c:func:`PyThread_create_key`: use :c:func:`PyThread_tss_alloc`
+  * :c:func:`PyThread_delete_key`: use :c:func:`PyThread_tss_free`
+  * :c:func:`PyThread_set_key_value`: use :c:func:`PyThread_tss_set`
+  * :c:func:`PyThread_get_key_value`: use :c:func:`PyThread_tss_get`
+  * :c:func:`PyThread_delete_key_value`: use :c:func:`PyThread_tss_delete`
+  * :c:func:`PyThread_ReInitTLS`: unneeded since Python 3.7
+
 Removed
```

Comment:
I would prefer to list Pending Removal at the end, after Removal which are done (where it was).

Reply:
IMO things which already happened should be documented before planned changes.

You change the order in 3.13 and leave 3.12 unchanged.